spacelift/internal/structs/module.go

```diff
@@ -43,9 +43,11 @@ func (m *Module) ExportVCSSettings(d *schema.ResourceData) error {
 		vcsSettings["project"] = m.Namespace
 		fieldName = "azure_devops"
 	case VCSProviderBitbucketCloud:
+		vcsSettings["id"] = m.VCSIntegration.ID
```

Comment:
ahh I missed it?

Reply:
I think so; looks like each VCS should have that.